src/main/proto/service.proto

```diff
@@ -50,4 +52,11 @@ message Account {
     FROZEN = 2;
   }
   optional Status status = 3;
-}
\ No newline at end of file
+}
+
+message AddAccountForCustomerRequest {
+  optional bytes customerId = 1;
+  optional bytes accountId = 2;
+  optional bytes roleId = 3;
```

Comment:
The role id is the unique identifier for the new role. I'd prefer if this is generated by the backend and returned in the response. (We should also return customer_id / account_id in the CreateX methods, but this can be done in a separate PR)

Reply:
i think ill do this in a separate PR because it means changing the signature in a lot of files, which might crowd this PR.